index.markdown

```diff
@@ -343,6 +345,21 @@ redirect_from: '/moveit/'
                   <div class="release-versions__3">STATUS</div>
                 </div>
                 <div class="release-versions__body">
+                  <div class="release-versions__content">
```

Comment:
I added Rolling with status "DEVELOPED" and Galactic as "RECOMMENDED". Do we even have a preference? Seems odd to list it like this if both are technically the same right now. In the future I'd say Rolling should be recommended for devs and Galactic for more long-term stability.

Reply:
Rolling won't be stable, so I think we should recommend Galactic (or whatever is the latest non-rolling relase) to the general user base.

I agree that Rolling is best for MoveIt developers, which is different from the general developer base. Notably, anyone who is downloading MoveIt is a developer.